### PR TITLE
Organizations

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

**Comment by DwayneJengSage** [comment]
Is .pom.xml.swp generated by the IDE or something? We should remove that from the repo, possibly add an entry to .gitignore.

File: src/test/java/org/sagebionetworks/bridge/services/OrganizationServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        existing.setCreatedOn(CREATED_ON);
+        when(mockDao.getOrganization(TEST_APP_ID, IDENTIFIER)).thenReturn(Optional.of(existing));
+        
+        when(mockDao.updateOrganization(org)).thenReturn(org);
+        
+        Organization retValue = service.updateOrganization(org);
+        assertEquals(retValue.getAppId(), TEST_APP_ID);
+        assertEquals(retValue.getIdentifier(), IDENTIFIER);
+        assertEquals(retValue.getName(), NAME);
+        assertEquals(retValue.getCreatedOn(), CREATED_ON);
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/services/OrganizationServiceTest.java:140]: Minor: Check modifiedOn.

File: src/test/java/org/sagebionetworks/bridge/validators/OrganizationValidatorTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import static org.sagebionetworks.bridge.TestConstants.TEST_APP_ID;
+import static org.sagebionetworks.bridge.TestUtils.assertValidatorMessage;
+import static org.sagebionetworks.bridge.validators.OrganizationValidator.INSTANCE;
+import static org.sagebionetworks.bridge.validators.Validate.CANNOT_BE_BLANK;
+
+import org.testng.annotations.Test;
+
+import org.sagebionetworks.bridge.models.organizations.Organization;
+
+public class OrganizationValidatorTest {
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/validators/OrganizationValidatorTest.java:14]: Add a case for a valid Organization.

**Review by DwayneJengSage — APPROVED** [review]
Looks good. 2 comments about tests.

File: src/test/java/org/sagebionetworks/bridge/validators/OrganizationValidatorTest.java (hunk 2 above)
Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/validators/OrganizationValidatorTest.java:14]: Added

File: src/test/java/org/sagebionetworks/bridge/services/OrganizationServiceTest.java (hunk 1 above)
Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/services/OrganizationServiceTest.java:140]: Added

**Comment by alxdarksage** [comment]
Removed the swap file (that's an artifact of having it open in Vim).

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**